lib/App/pmuninstall.pm

```diff
@@ -309,7 +309,8 @@ sub is_local_lib {
     return unless $self->{local_lib};
 
     my $local_lib_base = quotemeta File::Spec->canonpath(Cwd::realpath($self->{local_lib}));
-    $file = File::Spec->canonpath(Cwd::realpath($file));
+    $file = File::Spec->canonpath($file);
+    $file = Cwd::realpath($file);
```

Comment:
Yeah, this one feels off... even if it passes, switching the order of only one should not be a fix. It may work on any OS *other than Windows*... :-/